src/Home.js

```diff
@@ -62,30 +47,10 @@ export default class Home extends Component {
     window.location.hash = myParams.toString();
   }
 
-  setInitialParams() {
-    const myQueryString = window.location.hash.slice(1);
-    const myParams = new URLSearchParams(myQueryString);
-    myParams.set('page', this.state.page);
-    myParams.set('perPage', this.state.numberOfResultsPerPage);
-    window.location.hash = myParams.toString();
-  }
-
-  handleSubmit = (e) => {
-    const form = document.getElementById('myForm');
+  handleSubmit = async (e) => {
     e.preventDefault();
-    const formData = new FormData(form);
-
-    const myQueryString = window.location.hash.slice(1);
-    const myParams = new URLSearchParams(myQueryString);
-
-    //myParams.set('type', formData.get('type'));
-    const mySearch = formData.get('search');
-    myParams.set('pokemon', formData.get('search'));
-    console.log('mySearch', mySearch);
-    // Reset to page 1 as this is new search and
-    myParams.set('page', 1);
-
-    window.location.hash = myParams.toString();
+    const data = await request.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex?pokemon=${this.state.searchInput}`);
+    this.setState({pokeData: data.body.results});
```

Comment:
You're going to want to use history.push here similarly to `this.props.history.push(this.state.searchQuery)` — this will get your searchQuery into your URL